sds_data_manager/sds_data_manager_stack.py

```diff
@@ -31,6 +32,32 @@ def __init__(
             block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
         )
 
+        ########### CONFIG STORAGE
+        # This is the S3 bucket where the configurations will be stored
+        config_bucket = s3.Bucket(
+            self,
+            "CONFIG-BUCKET",
+            bucket_name=f"sds-config-{sds_id}",
+            versioned=True,
+            removal_policy=RemovalPolicy.DESTROY,
+            auto_delete_objects=True,
+            block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
+        )
+
+        s3_deploy.BucketDeployment(
+            self,
+            "DeployConfig",
+            sources=[
+                s3_deploy.Source.asset(
+                    os.path.join(
+                        os.path.dirname(os.path.realpath(__file__)),
+                        "config",
+                    )
```

Comment:
We should use pathlib to make our lives easier if possible: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module
`pyupgrade` may even handle this for you if you want to try adding that to the project too: https://github.com/asottile/pyupgrade

Reply:
Updated to use pathlib. I didn't want to try adding the pyupgrade now since the pre-commit is still failing

Reply:
Sounds good, I agree with holding off for now. But, you should try and get pre-commit passing here still. It looks like running black again will possibly fix it looking at the pre-commit error log.

You should be able to install pre-commit locally so it will fix those automatically for you as well if you want.

Reply:
Ah ok, I didn't realize that was working now. I'll fix those issues

Reply:
You can try adding a message `pre-commit.ci autofix` as a comment on this PR for it to do the work for you too if it is an easy fix.